__tests__/html2/hooks/useActiveTyping.variable.html

```diff
@@ -78,10 +77,7 @@
         // 3. t=10: Typed letter "b"
         // 4. t=10: Scheduled another typing indicator at t=3000
         // 5. t=3000: Send typing activity
-        clock.tick(3000);
-
-        // TODO: Understand why we need to yield after clock.tick and see if we could eliminate it.
-        await 0;
+        await directLine.actPostActivity(() => ponyfill.tick(3000));
```

Comment:
Did you understood?

Reply:
Nope. But after I move from DLJS to DL Emulator, it is no longer needed. If I keep it, it will add 10ms to the test case.